src/plone/restapi/tests/http-examples/transactions_revert.req

```diff
@@ -0,0 +1,9 @@
+PATCH /plone/@transactions HTTP/1.1
+Accept: application/json
+Authorization: Basic YWRtaW46c2VjcmV0
+Content-Type: application/json
+
+{
+    // One or more Transaction_Id can be inserted in the List
```

Comment:
This is json. No comments allowed.
With the comment removed the build of the docs will work correctly.

Reply:
Done @ksuess.